.rubocop.yml

```diff
@@ -645,7 +644,7 @@ Style/BlockComments:
 Style/BlockEndNewline:
   Description: Put end statement of multiline block on its own line.
   Enabled: true
-Style/Blocks:
+Style/BlockDelimiters:
```

Comment:
This one fixes this warning
Warning: unrecognized cop Style/Blocks found in /Users/a/Desktop/westmill/simple_loads/.rubocop.yml
which was introduced in rubocop 0.30.0
The Gemfile.lock specifies 0.34.2